UI/test/integration-tests/speedLimitVisualizationSpec.js

```diff
@@ -23,13 +23,13 @@ define(['chai', 'testHelpers'], function(chai, testHelpers) {
     });
     it('displays speed limits', function() {
       var speedLimitVectors = lineStrings(testHelpers.getSpeedLimitFeatures(openLayersMap));
-      var limits = _.map(speedLimitVectors, function(vector) { return vector.getProperties().value; });
+      var limits = _.map(speedLimitVectors, function(vector) { return vector.getProperties().value.value; });
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code